src/components/pacbio/PacbioLibraryForm.vue

```diff
@@ -179,40 +168,15 @@ formLibrary is a reactive variable, so it will update when the library prop chan
 initialize formLibrary with the library prop
 */
 const formLibrary = ref(props.library ? { ...props.library } : {})
+defineExpose({ formLibrary })
 
-/*
-Expose formLibrary to parent component as a reactive variable,
-so that the parent component can access the formLibrary and its properties
-*/
```

Comment:
Quite a lot documentation removed in a couple of places. It would be good to keep those.